src/providers/export/tensorFlowPascalVOC.ts

```diff
@@ -64,71 +85,92 @@ export class TFPascalVOCJsonExportProvider extends ExportProvider<ITFPascalVOCJs
         await this.storageProvider.createContainer(jpegImagesFolderName);
 
         const allImageExports = allAssets.map((element) => {
+            return this.exportSingleImage(jpegImagesFolderName, element);
+        });
+
+        try {
+            await Promise.all(allImageExports);
+        } catch (err) {
+            console.log(err);
+        }
+    }
+
+    private async exportSingleImage(jpegImagesFolderName: string, element: IAssetMetadata): Promise<void> {
```

Comment:
This method is pretty large - I recommend you break this down into smaller easier to digest functions.